_posts/2017-09-19-l-intrapreneur-n-est-pas-un-product-owner.md

```diff
@@ -66,7 +68,7 @@ Le porteur de la Startup d’État est plus proche d’un entrepreneur que d’u
 
 Ça y est, on a trouvé. C’est un intrapreneur de Startup d’État !
 
-Le terme intrapreneur ne tire pas les connotations restrictives de PO et englobe plus que les missions du chef de produit. Bien sûr, le terme deviendra beaucoup moins sexy dans 5 ans, quand des intrapreneurs de tout poil auront envahi les organisations, et que le terme sera devenu galvaudé et chargé de connotations négatives 😉
+Le terme intrapreneur ne tire pas les connotations restrictives de PO et englobe plus que les missions du chef de produit. Bien sûr, le terme deviendra beaucoup moins sexy dans 5 ans, quand des intrapreneurs de tout poil auront envahi les organisations, et que le terme sera devenu galvaudé ... à moins que nous gardions bien en tête la définition suivante : celui qui se revendique intrapreneur doit fournir un URL. 
```

Comment:
"fournir un URL" c'est pas super clair...

Reply:
u see it u own it ;)

Reply:
"doit avoir un ou plusieurs produits numériques dans sa vitrine, et des clients satisfaits" ?